app/views/dashboard/accounts/settings.html.erb

```diff
@@ -11,9 +11,44 @@
           <% else %>
             <%= render "subscription_management" %>
           <% end %>
+        <% end %>        
+
+        <h2 class="mt-6">Sync mode (advanced feature)</h2>
```

Comment:
(advanced) is enough in the title. Can remove 'feature'

Reply:
Fixed in 86d8dce